packages/react-components/react-toast/src/state/types.ts

```diff
@@ -17,9 +17,16 @@ export interface ToastOptions {
   dispatchedAt: number;
 }
 
+export interface ToastOffsetObject {
+  horizontal?: number;
+  vertical?: number;
+}
+
+export type ToastOffset = Partial<Record<ToastPosition, ToastOffsetObject>> | ToastOffsetObject;
```

Comment:
```suggestion
export type ToastOffset = Partial<Record<ToastPosition, ToastOffsetObject | number>> | ToastOffsetObject | number;
```

Should it be possible to define `offset` by a single number?

Reply:
I don't think there's any axis that's better than the other in this case. In any case if we need a number shorthand that can be added without breaking API